README.md

```diff
@@ -13,7 +13,7 @@ The library of UI components used for Terminus applications.
 [![Github release][gh-release-badge]][gh-releases]
 [![Library size][file-size-badge]][raw-distribution-js]
 <br>
-[![components](https://img.shields.io/badge/Components-35-%2316a085.svg)](#components)
+[![components](https://img.shields.io/badge/Components-38-%2316a085.svg)](#components)
```

Comment:
I thought I have seen other pr that had 37, so thinking this pr might just be behind. Will get this resolved once rebase to release. I'm purposely delaying rebase because want to get this approved first, then squash, rebase once to make final cut...

Reply:
Yeah rebasing sooner than later is probably good cause we will still need to look it over again after the rebase.